tokio/src/time/interval.rs

```diff
@@ -1,17 +1,20 @@
 use crate::future::poll_fn;
 use crate::time::{sleep_until, Duration, Instant, Sleep};
 
-use std::future::Future;
 use std::pin::Pin;
 use std::task::{Context, Poll};
+use std::{convert::TryInto, future::Future};
 
-/// Creates new `Interval` that yields with interval of `duration`. The first
-/// tick completes immediately.
+/// Creates new [`Interval`] that yields with interval of `period`. The first
+/// tick completes immediately. The [`MissedTickBehavior`] is the
+/// [default one](MissedTickBehavior::default), which is
+/// [`Burst`](MissedTickBehavior::Burst).
```

Comment:
```suggestion
/// Creates new [`Interval`] that yields with interval of `period`. The first
/// tick completes immediately. The default [`MissedTickBehavior`] is
/// [`Burst`](MissedTickBehavior::Burst), but this can be configured 
/// by calling `set_missed_tick_behavior`.
```